plugins/nf-sqldb/build.gradle

```diff
@@ -13,11 +13,59 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+plugins {
+    id 'de.undercouch.download' version '4.1.2'
+}
+
+/**
+ * The following tasks download and confirm the MD5 checksum of the ZIP archive
+ * for Simba Athena JDBC driver and extract its contents to the build directory
+ */
+task downloadZipFile(type: Download) {
+    src 'https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC-2.0.25.1001/SimbaAthenaJDBC-2.0.25.1001.zip'
+    dest new File(buildDir, 'SimbaAthenaJDBC-2.0.25.1001.zip')
+}
+
+task verifyFile(type: Verify, dependsOn: downloadZipFile) {
+    src new File(buildDir, 'SimbaAthenaJDBC-2.0.25.1001.zip')
+    algorithm 'MD5'
+    checksum '71504d0317656d790978363358d0c068'
+}
+
+task downloadAndUnzipFile(dependsOn: [downloadZipFile, verifyFile] , type: Copy) {
+    from zipTree(downloadZipFile.dest)
+    into "${projectDir}/libs"
+}
+
+task copyAthenaSimbaJar(dependsOn: downloadAndUnzipFile, type: Copy) {
+    from "${projectDir}/libs/SimbaAthenaJDBC-2.0.25.1001/AthenaJDBC42_2.0.25.1001.jar"
+    into "${projectDir}/libs"
+}
+
+task downloadAndCleanUpLibsFolder(dependsOn: copyAthenaSimbaJar, type: Delete) {
+    delete "${projectDir}/libs/SimbaAthenaJDBC-2.0.25.1001/"
+}
+
+defaultTasks 'downloadAndCleanUpLibsFolder'
+
+build.dependsOn downloadAndCleanUpLibsFolder
+compileGroovy.dependsOn downloadAndCleanUpLibsFolder
+compileTestGroovy.dependsOn downloadAndCleanUpLibsFolder
+test.dependsOn downloadAndCleanUpLibsFolder
+copyPluginLibs.dependsOn downloadAndCleanUpLibsFolder
```

Comment:
There must be a better way to do this no? 🤔

Reply:
Paolo, as per you comment https://github.com/nextflow-io/nextflow/pull/2500#issuecomment-996916180 , I've added `slf4j:log4j-over-slf4j` library, but not sure how exactly to configure it.

Reply:
Yeah, a bit ugly. Should not be enough 

```
compileGroovy.dependsOn downloadAndCleanUpLibsFolder
``` 

and then it will pull the parent tasks?

Reply:
Unfortunately, for this one, if I only include `compileGroovy.dependsOn ...` it fails to compile and it doesn't trigger the download and import of the JAR file.

```
Execution failed for task ':plugins:nf-sqldb:copyPluginLibs'.
> Could not resolve all files for configuration ':plugins:nf-sqldb:runtimeClasspath'.
   > Could not find :AthenaJDBC42_2.0.25.1001:.
     Searched in the following locations:
       - file:/Users/eklavya/projects/code/nextflow/plugins/nf-sqldb/libs/AthenaJDBC42_2.0.25.1001.jar
     Required by:
         project :plugins:nf-sqldb

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 57s

```


The other task dependencies I added because the Gradle build system threw a warning that the build might not be reproducible if I don't add these dependencies.